app/controllers/subscribers_from_account_token_controller.rb

```diff
@@ -0,0 +1,79 @@
+class SubscribersFromAccountTokenController < ApplicationController
```

Comment:
There seems to be a heck of a lot going on in a controller here which looks like a bit of a smell. Would we be able to abstract the account API client to a separate class? To do so you can probably trim a bunch of boilerplate by having it inherit from GdsApi::Base.

Reply:
With the switch to checking a JWT's signature, the controller is much smaller now.